# Implements SBP MSG_ANGULAR_RATE

uint32 tow    # GPS Time of Week
int32 x       # angular rate about x axis
int32 y       # angular rate about y axis
int32 z       # angular rate about z axis
uint8 flags   # Status ï¬‚ags (See SBP manual for details)
